tests/conftest.py

```diff
@@ -70,3 +70,16 @@ class ThreadingHTTPServer(ThreadingMixIn, HTTPServer):
         server_thread.start()
         yield server.server_address
         server.shutdown()
+
+
+@pytest.fixture(scope="function", autouse=True)
+def clear_lightning_env_variables():
+    """ Runs before every test and deletes all PL specific env variables. """
+    blacklist = [key for key in os.environ.keys() if key.startswith("PL_")]
+    blacklist += [
+        "LOCAL_RANK",
+        "GLOBAL_RANK",
+        "WORLD_SIZE",
+    ]
+    for key in blacklist:
```

Comment:
```suggestion
    blocklist = [key for key in os.environ.keys() if key.startswith("PL_")]
    blocklist += [
        "LOCAL_RANK",
        "GLOBAL_RANK",
        "WORLD_SIZE",
    ]
    for key in blocklist:
```

https://github.com/pytorch/pytorch/issues/41443